source/Octopus.Client/Octopus.Client.nuspec

```diff
@@ -24,14 +24,15 @@
             </group>
         </dependencies>
         <frameworkAssemblies>
-            <frameworkAssembly assemblyName="System.ComponentModel.DataAnnotations" targetFramework=".NETFramework4.5" />
-            <frameworkAssembly assemblyName="System.Net.Http" targetFramework=".NETFramework4.5" />
-            <frameworkAssembly assemblyName="System.Numerics" targetFramework=".NETFramework4.5" />
+            <frameworkAssembly assemblyName="System.ComponentModel.DataAnnotations" targetFramework=".NETFramework4.5.2" />
+            <frameworkAssembly assemblyName="System.Net.Http" targetFramework=".NETFramework4.5.2" />
+            <frameworkAssembly assemblyName="System.Numerics" targetFramework=".NETFramework4.5.2" />
         </frameworkAssemblies>
     </metadata>
     <files>
-        <file src="bin\release\net45\Octopus.Client.dll" target="lib\net45" />
-        <file src="bin\release\net45\Octopus.Client.xml" target="lib\net45" />
+        <file src="icon.png" target="images\" />
```

Comment:
Is this needed @michaelnoonan ?

Reply:
Yes. It's related to `iconUrl` being deprecated. Embedding an image into the package is the recommended approach.